src/inmanta/main.py

```diff
@@ -294,8 +294,13 @@ def save_config(client: Client, env: Dict[str, str]):
         "host": client.host,
         "port": client.port,
     }
-    if os.path.exists(".inmanta"):
-        click.echo(".inmanta exists, not writing config", err=True)
+
+    if os.path.exists(".inmanta") and not click.confirm(".inmanta exits, do you want to overwrite it?"):
```

Comment:
This code looks really strange. It will ask for a confirmation twice. Let's add a test case.

Reply:
resolved. I added a test and cleaned up up the rest.